inference-engine/thirdparty/clDNN/kernel_selector/core/common/jitter.h

```diff
@@ -308,19 +308,21 @@ class FusedOpsCodeGenerator {
     struct idx_desc {
         std::string b;
         std::string f;
+        std::string w;
         std::string z;
         std::string y;
         std::string x;
         size_t dims;
         explicit idx_desc(std::vector<std::string> idx, DataTensor t)
-            : b("0"), f("0"), z("0"), y("0"), x("0"), dims(0) {
+            : b("0"), f("0"), w("0"), z("0"), y("0"), x("0"), dims(0) {
             dims = idx.size();
             switch (dims) {
                 case 1: f = idx[0]; break;
                 case 2: b = idx[0]; f = idx[1]; break;
                 case 3: b = idx[0]; f = idx[1]; y = idx[2]; break;
                 case 4: b = idx[0]; f = idx[1]; y = idx[2]; x = idx[3]; break;
                 case 5: b = idx[0]; f = idx[1]; z = idx[2]; y = idx[3]; x = idx[4]; break;
+                case 6: b = idx[0]; f = idx[1]; w = idx[2]; z = idx[3]; y = idx[4]; x = idx[5]; break;
                 default: throw std::runtime_error("More than 5 dimenstions is not supported in fused op generator");
```

Comment:
I think that we should update this exception message.

Reply:
Fixed